<?xml version="1.0" encoding="UTF-8"?><testrun name="PlayingCardTest" project="SEF" tests="1" started="1" failures="0" errors="1" ignored="0">
  <testsuite name="sef.module4.activity.PlayingCardTest" time="0.0">
    <testcase name="testPlayingCard" classname="sef.module4.activity.PlayingCardTest" time="0.0">
      <error>java.lang.IndexOutOfBoundsException: Index: 4, Size: 4&#13;
	at java.util.ArrayList.rangeCheck(Unknown Source)&#13;
	at java.util.ArrayList.get(Unknown Source)&#13;
	at sef.module4.activity.PlayingCard.toString(PlayingCard.java:162)&#13;
	at sef.module4.activity.PlayingCardTest1.testPlayingCard(PlayingCardTest1.java:18)&#13;
	at sef.module4.activity.PlayingCardTest.testPlayingCard(PlayingCardTest.java:10)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)&#13;
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)&#13;
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)&#13;
	at java.lang.reflect.Method.invoke(Unknown Source)&#13;
	at junit.framework.TestCase.runTest(TestCase.java:164)&#13;
	at junit.framework.TestCase.runBare(TestCase.java:130)&#13;
	at junit.framework.TestResult$1.protect(TestResult.java:106)&#13;
	at junit.framework.TestResult.runProtected(TestResult.java:124)&#13;
	at junit.framework.TestResult.run(TestResult.java:109)&#13;
	at junit.framework.TestCase.run(TestCase.java:120)&#13;
	at junit.framework.TestSuite.runTest(TestSuite.java:230)&#13;
	at junit.framework.TestSuite.run(TestSuite.java:225)&#13;
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:131)&#13;
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)&#13;
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)&#13;
&#13;
</error>
    </testcase>
  </testsuite>
</testrun>
